lib/functions/single_illust.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+var common = require("./common");
+
+var getillustUrlFromImage = function (img) {
+    return img.dataset["src"];
```

Comment:
__[eslint]__
*  Error - ["src"] is better written in dot notation. (dot-notation)